elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/schema/dimension/impl/TimeDimension.java

```diff
@@ -104,7 +111,9 @@ public int hashCode() {
     public String toString() {
         return new StringJoiner(", ", TimeDimension.class.getSimpleName() + "[", "]")
                 .add("timeZone=" + getTimeZone().getDisplayName())
-                .add("timeGrain=" + getTimeGrain())
+                .add("timeGrains=" + Arrays.stream(getSupportedGrains())
+                        .map(TimeGrainDefinition::grain)
+                        .collect(Collectors.toSet()))
```

Comment:
Should we just use a `Set` to store supportedGrains in the first place?

Reply:
:+1:

Reply:
We can remove the stream here I think.

Reply:
It is mapping between a TimeGrainDefinition to a TimeGrain - so we can't.